cob_generic_can/package.xml

```diff
@@ -13,6 +13,7 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <depend>cmake_modules</depend>
```

Comment:
see comment above!
Simply add `<depend>boost</depend>` instead of `<depend>cmake_modules</depend>`